### PR TITLE
adding a folder and template for product requirement documents

### PR DESCRIPTION
**Pull request by iglesiasbrandon**
What: adding a folder and template for product requirement documents

Why: so that we can start to post our PRDs on github

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/main/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/main/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

**Comment by elek** [comment]
Love it.

One comment (slightly unrelated).

Kubernetes uses a separated [repository](https://github.com/kubernetes/enhancements) for proposals. I like that pattern because it makes easier to track the open proposals, makes it possible to use custom PR templates and provides better visibility.

If PRD docs will be included in the doc folder (:heart:) we may have more content,  which may make it more reasonable to move PRD + old design docs to a dedicated repo.

File: docs/productrequirementdocuments/product-requirements-document-template.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Product Requirements Document Template
+
+
+## Objective
+
+Explain the customer problem we are solving and how it relates to our vision, goals, and initiatives. This will establish the high-level purpose of what we want to accomplish and for whom. This also keeps the broader team focused on building features that delight our customers — resulting in us delivering a MLP
```
Comment (ifraixedes) [inline docs/productrequirementdocuments/product-requirements-document-template.md:6]: What does MLP stand for?

**Comment by iglesiasbrandon** [comment]
@elek we could do that, I don't have a preference on where the PRDs live as long as they are in a public repo! I added them here because we had the blueprints and testplans. We could bring this up in Arch review today and get feedback from the team!

File: docs/productrequirementdocuments/product-requirements-document-template.md (hunk 1 above)
Comment (iglesiasbrandon) [reply docs/productrequirementdocuments/product-requirements-document-template.md:6]: Minimum Lovable Product but im going to remove that bit.

**Comment by iglesiasbrandon** [comment]
going to close this PR, we are going to open PRDs in the roadmap repository for now.

**Pull request closed without merging**